include/ignition/gazebo/ServerConfig.hh

```diff
@@ -36,6 +37,22 @@ namespace ignition
     // Forward declarations.
     class ServerConfigPrivate;
 
+    /// \brief Type of SDF source.
+    enum class SourceType
+    {
+      // The source is an SDF file.
+      kSdfFile,
+
+      // No source specified.
+      kNone,
```

Comment:
Is there a specific reason that kNone isn't first?

Reply:
I was going alphabetic order. I'll move it to first.

Reply:
Changed in [7d33cf0](https://github.com/ignitionrobotics/ign-gazebo/pull/1333/commits/7d33cf0aafacefa3a4bdaa1522ce02b7d7b5263b)